src/components/main_container/MapView.tsx

```
@@ -0,0 +1,199 @@
+            const geojsonObj = wktToGeoJSON(geoValue['@value']);
+            geoJson.push(
+              URL.createObjectURL(
+                new Blob([JSON.stringify(geojsonObj)], { type: 'application/json' })
+              )
+            );
+          } catch (e) {
+            console.warn('Failed to parse WKT:', geoValue['@value'], e);
+          }
+        }
+      }
+      continue;
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)